src/Controls/src/Core/VisualElement/VisualElement_StyleSheet.cs

```diff
@@ -8,6 +9,7 @@
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/VisualElement.xml" path="Type[@FullName='Microsoft.Maui.Controls.VisualElement']/Docs/*" />
+	[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields)]
```

Comment:
I'm thinking about putting CSS styling behind a feature flag instead of preserving all public and non-public fields on all `VisualElement` classes. Would it be acceptable to say that CSS isn't supported with trimming?

On the other hand, when I compared the .ipa size of NativeAOT app build with this `[DAM]` and without it, the size difference was negligible. I worry that when somebody uses Telerik or some other control library, it might suddenly become more pronounced.